### ISSUE TITLE
this is terrible

### ISSUE THREAD

**Issue by Gignoot** [tonioni/WinUAE#201]
Guys, can anyone explain to me why your winaeu  changes it's own settings behind the users back ??

I'd check your computers for viruses if i were you, cause this crap ain't working .

yeah sure a500,a1200 and cd32 work, but of all the others it works one time , then the next it doesn't, and so on and on and on.
I don't need this emulator for the 500 1200 or cd32....it's just the other things i wanted to have a look at but allas, it won't allow anything else really by constantly sabotaging itself,  do a few reboots and sometimes it works only to fail on the next boot...crap.
This is just crapware that acts like a virus on itself. After ALL those years ???? it's a joke.  Another MS  update and it won't work at all anymore.

**Comment by dirkwhoffmann** [comment]
@Gignoot, could you try to describe reproducible scenarios that exhibit the bugs? Otherwise, nobody will be able to help you.

Besides, I don’t think your judgement about UAE is fair. UAE is the most compatible Amiga emulator out there and many people have spent uncountable hours of their spare time on this project. Please be aware that writing an emulator of such complexity is anything but a cakewalk (just assume that I know what I am talking about).

**Comment by Gignoot** [comment]
I will try to describe it , but give me some time to properly construct it.

Sure UAE works very well in the easier standard situations, say like the Amiga 500 up to 4000, but when i try to do something more exotic like using ppc or even the 060, having the right roms, one boot it works, only to fail on the next with software or other errors , this is a bit of a pain when it isn’t consistent. 
Also many problems with HD’s , like getting them recognized,activated and partitioned for installation. One time it works , the next it doesn’t, and i have to quit winuae to be best between every virtual machine boot, and that only improves the problems slightly.  Sure it could be caused by windows 10 and all it’s little updates that continuesly change how windows handles “security”.  I don’t know if the amiga reported software fail codes during boot would be of any use to you ? Then there is also options in winuae that never got documented as to what they actually do, or why some are greyed out in certain situations, it’s a pain having to gamble and just test all possible itterations of config that way.

On the side i am also surprised to find that for cd32 emulation of playing Video-cd , i need the fmv module while my real cd32 doesn’t need that at all, it just plays them, with correct sound synchronization which winuae can’t seem to do either, sound is Always to early, no matter what delay gets set. The audio delay slider does actually nothing at all.

This is what i can already say about it in short.

Kind regards.



Van: Dirk Hoffmann
Verzonden: maandag 1 november 2021 16:20
Aan: tonioni/WinUAE
CC: Gignoot; Mention
Onderwerp: Re: [tonioni/WinUAE] this is terrible (Issue #201)

@Gignoot, could you try to describe reproducible scenarios that exhibit the bugs? Otherwise, nobody will be able to help you.
Besides, I don’t think your judgement about UAE is fair. UAE is the most compatible Amiga emulator out there and many people have spent uncountable hours of their spare time on this project. Please be aware that writing an emulator of such complexity is anything but a cakewalk (just assume that I know what I am talking about).
—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by tonioni** [comment]
"Not enough information" (and "buy a new PC" is the best answer to that kind of attitude) and because no one else has reported anything similar, there must be something else than simply being some weird WinUAE problem.

CD32 does need MPEG module to play VCDs. At least plain CD32 without any accelerators. (Many games use other methods for videos and work without MPEG module). And WinUAE is not a media player, video sync problem is normal and by design. (It is not trivial to get emulated video players work without glitches and this is the only part of emulation that would need it)

Also I don't do documentation. Use something else designed to be easy to use or some front end. (and check eab, lots more useful information there, github is not the best way to find out how WinUAE works)

Usual way to debug these kinds of problems is to run winuae with -log command line parameter (use windowed mode!), it shows log window and log data also goes to winuaelog.txt and then compared logs/check for possible unexpected errors etc. WinUAE by design does not open any dialogs if some file fails to open because it would be really annoying but they are added to GUI Error log window.

If config seems to change suddenly, make sure Quickstart mode is disabled.

**Comment by Gignoot** [comment]
Yeah probably all users who don’t use UEFI, nor VBS or any of that microsoft security shit. A new pc is not needed at all friend, but your documentation is severly lacking, a program that doesn’t come with what the installer shows of options. The installer offers to install the help files but they are not included.

If it has problems on a windows that has security running, that may be the reason some advice to run it in adminstrator mode, which by itself is a security risk i don’t even want to think about.

It’s very odd that a stock amiga is perfectly working but don’t try any of the options or it starts acting weird.  But ok, i’ll just use it then for what it can do.  I got the latest .chm file by now but that too wasn’t easy when there are two downloads, one has the chm the other are html files , but both are alluded to be the same contents. 
This is what i mean by being clearer with documenting, the help files are downloadable on win uae site , but i couldn’t find them with ease, i had to ask and then i downloaded the wrong one first containing the html files, not very clear. Remember that there may be new users who want to try it and they get left in the cold i’m sure.

The flawless emulation of cd32 video play back would have been very nice to review all my collected video-cd but bummer , i guess i’ll have to toss em after all. Though the sound sometimes starts out synced and at a point it glitches and sound starts running ahead.

So in al honesty, it still needs work (details). But does it even matter anymore , it’s good for nostalgia, could be good for teaching kids the first thing about computers.  Which i can advice it for , but nothing fancy.  And why do users need to find some firmwares by themselves ? You could just aswell include those too, or is there someone who owns them? Which i don’t think is possible. Or does cloanta think they own everything ? Do they hold the only licence perhaps ?  They ask money for using your software and the other emulator , do you get your share when you offer it as a free program ?

Maybe i’ll run the program through Ghidra if i feel like checking what’s inside, who knows i might find something out of order. In which case i will keep you updated , if you are actually interested in any help that is. I will try that -log option first.

Yes, quickstart is Always disabled., i have seen several YouTube vids on howto config for PPC but they all use different and older versions of winuae and use options that stay either greyed out or no longer there or replaced with other options in the latest version, this makes it very difficult.

Anyway, kind greetings

Van: Toni Wilen
Verzonden: donderdag 4 november 2021 19:39
Aan: tonioni/WinUAE
CC: Gignoot; Mention
Onderwerp: Re: [tonioni/WinUAE] this is terrible (Issue #201)

"Not enough information" (and "buy a new PC" is the best answer to that kind of attitude) and because no one else has reported anything similar, there must be something else than simply being some weird WinUAE problem.
CD32 does need MPEG module to play VCDs. At least plain CD32 without any accelerators. (Many games use other methods for videos and work without MPEG module). And WinUAE is not a media player, video sync problem is normal and by design. (It is not trivial to get emulated video players work without glitches and this is the only part of emulation that would need it)
Also I don't do documentation. Use something else designed to be easy to use or some front end. (and check eab, lots more useful information there, github is not the best way to find out how WinUAE works)
Usual way to debug these kinds of problems is to run winuae with -log command line parameter (use windowed mode!), it shows log window and log data also goes to winuaelog.txt and then compared logs/check for possible unexpected errors etc. WinUAE by design does not open any dialogs if some file fails to open because it would be really annoying but they are added to GUI Error log window.
If config seems to change suddenly, make sure Quickstart mode is disabled.
—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.
